### PR TITLE
Feature: Added app name to the splash screen

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

During this few months tiral of initial splash screen, user have never complained about it. However, we noticed the blurry of the image and then @mdtauk suggested great idea to add the build branch name under the splash screen image.

## Task Checklist

- [x] Fixed image blurry
- [x] Added the label of the build branch
- [x] Clean up
- [ ] Apply requested changes

## Known Issues

_N/A_

## PR Checklist

- [x] **Closes**: _N/A_
- [x] **Approval**: Have discussed with and got approval from the team[^1]
- [x] **Tests**: Tested accessibility on the local end
- [x] **Deployment**: Deployed the app on the local end
- [ ] **Localization**: Modified en-US string resources[^2]
- [x] **Co-authors**: @mdtauk

## Steps To Validate Changes

1. Launch the app
2. See the splash screen, which will be shown in second you launched and only for few moments

## Screenshots

image

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.

### CODE REVIEW

File: src/Files.App/Views/SplashScreenPage.xaml.cs [hunk 1]
```diff
@@ -11,17 +11,25 @@ namespace Files.App.Views
...
+				_ => string.Empty,
+			};
+
 		public SplashScreenPage()
 		{
 			InitializeComponent();
 		}
 
-		private void Image_ImageOpened(object sender, RoutedEventArgs e)
+		private void SplashScreenImage_ImageOpened(object sender, RoutedEventArgs e)
```
Comment (yaira2) [inline src/Files.App/Views/SplashScreenPage.xaml.cs:27]: Do we really need these events? I think @d2dyno1 suggested we remove them.

Comment (0x5bfa) [reply src/Files.App/Views/SplashScreenPage.xaml.cs:27]: I can remove. I don't know what is this TCS doing actually.

Comment (d2dyno1) [reply src/Files.App/Views/SplashScreenPage.xaml.cs:27]: I removed it in my PR

Comment (0x5bfa) [reply src/Files.App/Views/SplashScreenPage.xaml.cs:27]: Ok then, I’d like this to be merged.

Comment (d2dyno1) [reply src/Files.App/Views/SplashScreenPage.xaml.cs:27]: If you could rename it back to what it was, to avoid any future conflicts, if it's not needed 🙂

Comment (0x5bfa) [reply src/Files.App/Views/SplashScreenPage.xaml.cs:27]: Sure.

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**